viewsb/packet.py

```diff
@@ -756,7 +757,7 @@ def summarize(self):
 class USBControlTransfer(USBTransfer):
     """ Class representing a USB control transfer. """
 
-    FIELDS = {'request_type', 'recipient', 'request_number', 'value', 'index', 'request_length', 'stalled'}
+    FIELDS = {'sequence', 'request_type', 'recipient', 'request_number', 'value', 'index', 'request_length', 'stalled'}
```

Comment:
`sequence` is already added to the base class, so I don't think this is needed?

Reply:
Thanks for your review Mike. 

The sequence field on the USBControlTransfer class is required to be able to visualize the sequence number for transfers that get grouped; since this class does not inherit from ViewSBPacket.

Omission of this field will result in missing packet# for those groups:
![image](https://user-images.githubusercontent.com/1499454/150392886-dc904389-47bb-4456-af17-4367d926f477.png)


I tried to write my patch in such a way that the changes to the overall architecture are limited. If you want I can have a look at refactoring the packet classes/fields to have a single definition for common fields -- but feel this would make more sense as a separate change/PR. Please let me know what direction makes sense for you.

Reply:
Ah right, I see the issue. USBControlTransfer does inherit from ViewSBPacket (via USBTransfer), but you have to add 'sequence' to the list of fields to copy from the setup packet here: https://github.com/usb-tools/ViewSB/blob/3167a791162b8e9919035431a1dcc48d34f1a981/viewsb/packet.py#L764-L769

Reply:
Got it! updated my PR to add the new field to the list of fields to be copied (instead of re-defining it locally)

Reply:
Thanks!